### DISCUSSION TITLE
Stop logging to stdout (/var/log/messages)

### DISCUSSION THREAD

**Discussion by seanzechman** [cybertec-postgresql/pg_timetable#594] [Q&A] [answered]
We upgraded a while ago to the version that supported extra logging options like "--log-file=" (v5.5.0 specifically now) in hopes that timetable messages would stop appearing in /var/log/messages,  However, now it appears that we get messages in /var/log/messages and /var/log/pg_timetable.log file.  Is there a way to stop output from appearing in the /var/log/messages system log file?

Thank you so much for all your work!

**Comment by pashagolub** [comment, marked as answer]
Hello,

will `pg_timetable ... > /dev/null` solve your problem?

Log file is different from what user might see on the screen

**Reply by seanzechman** [reply]
Yes that works - I added 2>&1 to the end as well to also send errors to null.  Thanks and I should have tried that to begin with but I guess I thought if you used the logfile option it would skip the system log

**Reply by pashagolub** [reply]
Frankly speaking, we discussed that. The main difference is `stdout` is colorized and human readable. It's for real time ~meditation~ observation. :)

On the other hand file logs are for future analysis. That's why it JSON by default and can be splitted into chunks, and rotations, and so on, and so on...
